must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -114,3 +113,23 @@ echo "collecting dump of oc get obc all namespaces" | tee -a  "${BASE_COLLECTION
 echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplication --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vr_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect volumereplication --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For StorageSystem of all namespaces
+echo "collecting dump of storagesystems across all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get storagesystem --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/storagesystem_all_ns"
+{ oc describe storagesystem --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/storagesystem_describe_all_ns"
+{ oc get storageclusters --all-namespaces -o yaml; } >> "${BASE_COLLECTION_PATH}/namespaces/all/storagesystem_yaml_all_ns"
```

Comment:
Part of this is duplicating something we're already doing above: https://github.com/red-hat-storage/ocs-operator/pull/1328/files#diff-22a4750fbab2abc8b4e080042968dc784aac21fe537db2e4089618d828efe866R91

Also, I don't remember why we're not just doing an `inspect` yet. Can you try just adding storageclusters and storagesystems to the `resources` array and see if it works?

Reply:
I used inspect but it failed to log the YAML files but will try it out.

Reply:
Hm, then the inspect problem isn't resolved yet, it sounds like. We should investigate this further. ...later. In any case, the main thing that matters is consistency. At present we don't need to worry about our resources outside the openshift-storage Namespace. As such, i suggest doing the StorageSystem get above where the current StorageCluster get is. There's no need to do a describe on them at this point if we aren't doing it for StorageClusters.

Reply:
@umangachapagain suggested to add the describe command, but if it is not required, I shall remove it. Will do the changes as suggested by you and update the PR. Thanks @jarrpa

Reply:
> Hm, then the inspect problem isn't resolved yet, it sounds like. We should investigate this further. ...later. In any case, the main thing that matters is consistency. At present we don't need to worry about our resources outside the openshift-storage Namespace. As such, i suggest doing the StorageSystem get above where the current StorageCluster get is. There's no need to do a describe on them at this point if we aren't doing it for StorageClusters.

Just a question regarding the above suggestion: 
1. Storagecluster is only for openshift namespace, but for storagesystem we are collecting logs for all the namespace right?
2. Do we need to collect logs for both `oc get storageclusters --all-namespaces -o yaml` and `oc get storagesystem --all-namespaces` cause I don't see the same for storagecluster.